src/RealityData.ts

```diff
@@ -181,20 +181,23 @@ export class ITwinRealityData implements RealityData {
       const blobUrlRequiresRefresh = !containerCache?.timeStamp || (Date.now() - containerCache?.timeStamp.getTime()) > 3000000; // 3 million milliseconds or 50 minutes
 
       if (undefined === containerCache?.url || blobUrlRequiresRefresh) {
+        
+        const url = new URL(`${this.client.baseUrl}/${this.id}/${ writeAccess === true ? "writeAccess" : "readAccess"}`);
 
-        const url = this.iTwinId ? `${this.client.baseUrl}/${this.id}/container/?projectId=${this.iTwinId}&access=${access}`
-          : `${this.client.baseUrl}/${this.id}/container/?&access=${access}`;
-        const requestOptions = getRequestConfig(accessTokenResolved, "GET", url, this.client.apiVersion);
+        if(this.iTwinId)
+          url.searchParams.append("iTwinId", this.iTwinId);
 
-        const response = await axios.get(url, requestOptions);
+        const requestOptions = getRequestConfig(accessTokenResolved, "GET", url.href, this.client.apiVersion);
```

Comment:
where does the version come from?

Reply:
right now it's an enum with one value : `ApiVersion.v1` which is also default.
it's used in the `getRequestConfig` function in `RequestOption.ts` to set up the "accept":"application/vnd.bentley.itwin-platform.v1+json" header that is required.